vector/src/main/java/im/vector/app/features/home/HomeDetailViewModel.kt

```diff
@@ -241,15 +241,15 @@ class HomeDetailViewModel @AssistedInject constructor(
                                         roomSummaryQueryParams {
                                             memberships = listOf(Membership.INVITE)
                                             roomCategoryFilter = RoomCategoryFilter.ONLY_DM
-                                            activeSpaceFilter = activeSpaceRoomId?.let { ActiveSpaceFilter.ActiveSpace(it) } ?: ActiveSpaceFilter.None
+                                            spaceFilter = activeSpaceRoomId?.let { SpaceFilter.ActiveSpace(it) }
                                         }
                                 ).size
 
                                 roomsInvite = session.roomService().getRoomSummaries(
                                         roomSummaryQueryParams {
                                             memberships = listOf(Membership.INVITE)
                                             roomCategoryFilter = RoomCategoryFilter.ONLY_ROOMS
-                                            activeSpaceFilter = ActiveSpaceFilter.ActiveSpace(groupingMethod.spaceSummary?.roomId)
+                                            spaceFilter = groupingMethod.spaceSummary?.roomId?.let { SpaceFilter.ActiveSpace(it) } ?: SpaceFilter.OrphanRooms
```

Comment:
This code is equivalent (I think) to the previous code, hence, I am not sure why we would like to have `OrphanRooms` here. I am tempted to remove the fallback and pass `null` instead. WDYT @BillCarsonFr ?
Applicable to all the equivalent change in this file (and in other files too).

Reply:
could reduce some of the noise by extracting a `fun GroupingMethod.activeSpaceOrNull() / orOphans` extension

Reply:
Good idea, will do.

Reply:
c9a292c73b20fcb825e7a2fd5e27b6986f6b2d1d